flink-docs/src/main/java/org/apache/flink/docs/util/ConfigurationOptionLocator.java

```diff
@@ -73,15 +73,6 @@ public class ConfigurationOptionLocator {
                 new OptionsClassLocation("flink-clients", "org.apache.flink.client.cli"),
                 new OptionsClassLocation(
                         "flink-table/flink-sql-client", "org.apache.flink.table.client.config"),
-                new OptionsClassLocation(
-                        "flink-connectors/flink-connector-pulsar",
-                        "org.apache.flink.connector.pulsar.common.config"),
-                new OptionsClassLocation(
-                        "flink-connectors/flink-connector-pulsar",
-                        "org.apache.flink.connector.pulsar.source"),
-                new OptionsClassLocation(
-                        "flink-connectors/flink-connector-pulsar",
-                        "org.apache.flink.connector.pulsar.sink"),
```

Comment:
The associated generated files should be removed. The pulsar docs will have to provide equivalent files _somehow_.
(medium-term we need to make the config docs generator available to connector repos)

Reply:
I would think that right now we should 'just' remove it here and create a follow-up ticket for a permanent solution, WDYT?

Reply:
> The pulsar docs will have to provide equivalent files somehow.

Can we tackle that as part of a follow-up ticket?

Reply:
Did we create the follow-up ticket?

Reply:
It is now in FLINK-30486